server/graphql/v2/query/MemberInvitationsQuery.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLInt, GraphQLList } from 'graphql';
+
+import models from '../../../models';
+import { Forbidden, ValidationFailed } from '../../errors';
+import { MemberInvitation } from '../object/MemberInvitation';
+
+const MemberInvitationsQuery = {
+  type: new GraphQLList(MemberInvitation),
+  description: '[AUTHENTICATED] Returns the pending invitations',
+  args: {
+    CollectiveId: { type: GraphQLInt },
+    MemberCollectiveId: { type: GraphQLInt },
```

Comment:
"Collective" is not used anymore as a term on GQLV2. Also, we normally use the `AccountReferenceInput` type in these to provide more flexibility in the usage.

Reply:
Updated 👍🏻